modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -165,7 +165,7 @@ def buildProject(Map options = [:]) {
     lock("$repoName-$NODE_NAME-test") {
       if (hasDatabase()) {
         stage("Set up the database") {
-            runRakeTask("db:drop db:create db:schema:load")
+            runRakeTask("db:drop db:create db:schema:load_if_ruby db:structure:load_if_sql")
```

Comment:
I've always assumed `db:seed` to be used for bootstrap data in dev and prod, but not in test and managed that kind of thing on a per test basis with factories, or across the suite with fixtures.  I'm not entirely sure what we do here, but as we weren't previously using `db:seed` in this Jenkins script I assume it's more like that.

That said, we should probably get more eyes on this though as we don't want to go against the prevailing wind wrt. Rails conventions.